images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -45,9 +45,9 @@ bash.exe -c "pacman-key --init 2>&1"
 Write-Host "bash pacman-key --populate msys2"
 bash.exe -c "pacman-key --populate msys2 2>&1"
 
-Write-Host "pacman --noconfirm -Sy pacman"
-pacman --noconfirm -Sy pacman
-pacman --noconfirm -Su
+Write-Host "pacman -Sy --noconfirm --needed pacman"
+pacman -Sy --noconfirm --needed pacman
```

Comment:
As discussed in https://github.com/actions/virtual-environments/pull/906#issuecomment-632409785, pacman should NOT be updated first.

Reply:
@eine, Thank you for providing details. I think we can include these changes to the PR.

Reply:
@maxim-lobanov ^